src/EditModel/EditDataEntryForm/clampPaletteWidth.js

```diff
@@ -0,0 +1 @@
+export const clampPaletteWidth = width => Math.min(750, Math.max(width, 250))
```

Comment:
This function was moved to a separate file so that it could be used by both the `EditDataEntryForm` component and the `useResize` hook